client/src/styles/Icons.css

```diff
@@ -0,0 +1,74 @@
+@import url('./index.css');
+.bookmark__icon.solid {
+    position: relative;
+    fill: royalblue;
+}
+
+.like__icon.solid {
+    position: relative;
+    fill: rgb(148, 65, 225);
+}
+
+.log__icon.solid {
+    position: relative;
+    fill: limegreen;
+}
+
+.circular-icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 50px; /* adjust as needed */
+    height: 50px; /* adjust as needed */
+    border-radius: 50%;
+    /* background-color: #ccc; adjust as needed */
+}
+
+/* navbar icons */
+.nav__icon.home {
+    fill:#33b45e;
+}
+.nav__icon.bkmark {
+    fill: royalblue;
+}
+.nav__icon.log {
+    fill: limegreen;
+}
+.nav__icon.pop {
+    fill: #ff6600;
+}
+
+.product__icon {
+    width: 18pt;
+    margin: 4pt auto 2pt;
+}
```

Comment:
Please move this to `ProductCard.css` - Not the cleanest option but will at least avoid duplicate code for now